core/amber/src/main/python/core/runnables/main_loop.py

```diff
@@ -138,10 +137,10 @@ def process_control_payload(
         """
         start_time = time.time_ns()
         match(
-            (tag, get_one_of(payload)),
-            typing.Tuple[ActorVirtualIdentity, ControlInvocationV2],
+            (tag, get_one_of(payload, False)),
```

Comment:
what is this False? add key name to the variable, especially for boolean values.

Maybe it just needs a falsy value, how about None?

`get_one_of` is implemented by us. if we don't need the second parameter, we can remove it.

Reply:
now ControlPayloadV2 is not a `sealed_value` so we need to set `sealed=False`, added key name.